.circleci/config.yml

```diff
@@ -0,0 +1,140 @@
+version: 2.1
+
+orbs:
+  gem: doximity/gem-publisher@0
+
+executors:
+  ruby-latest:
+    resource_class: small
+    docker:
+      - image: cimg/ruby:2.7
+        environment:
+          - BUNDLE_VERSION: "~> 1.17"
+          - NEO4J_URL: "http://127.0.0.1:7474"
+      - image: neo4j:4.3.9-enterprise
+        environment:
+          NEO4J_ACCEPT_LICENSE_AGREEMENT: "yes"
+          NEO4J_AUTH: "none"
+          NEO4J_dbms_allow__upgrade: "true"
+          NEO4J_dbms_jvm_additional: "-Xms256m -Xmx256m"
```

Comment:
Do we want this set at 256? I don't know what we had before but this won't let any memory grow

Reply:
I think i copied this from what we have in the doximity circleci config